toponetx/datasets/mesh.py

```diff
@@ -36,14 +35,14 @@ def stanford_bunny(complex_type="simplicial complex"):
     ValueError
         If complex_type is not one of the supported values.
     """
-    if complex_type == "simplicial complex":
+    if complex_type == "simplicial":
         cpx = SimplicialComplex.load_mesh(DIR + "/bunny.obj")
         return cpx
-    elif complex_type == "cell complex":
+    elif complex_type == "cell":
```

Comment:
Do not use "cpx" for complex, write "complex" instead because it is not much longer and it helps code readability for contributors.

Also, replace "elif" by "if" because: the if-statement above ends with a "return", thus elif is not needed.

Reply:
I am not using complex because it is a keyword in Python. for complex numbers.

Reply:
I am not sure about your second comment, the else statement seems to be needed for me to take care of invalid input other than simplicial or cell, no?